CONTRIBUTING.md

```diff
@@ -41,84 +38,99 @@ Before submitting a bug, try to recreate it and document the steps we can take t
+If you want to find other feature requests open for upvoting, check out our [feature request view sorted by status](https://github.com/orgs/uswds/projects/8/views/18?sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=Status).
 
 #### 2. Submit an issue
 
-If your idea is not in the [current issues backlog](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22), submit an issue using the [feature request template](https://github.com/uswds/uswds/issues/new?assignees=&labels=Type%3A+Feature+Request%2CStatus%3A+Triage&template=feature_request.yaml&title=USWDS+-+Feature%3A+%5BYOUR+TITLE%5D).  A USWDS core team member may reach out to you if we need further clarification on your submission. 
+If your idea is not in the [current issues backlog](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22), submit an issue using the [feature request template](https://github.com/uswds/uswds/issues/new?assignees=&labels=Type%3A+Feature+Request%2CStatus%3A+Triage&template=feature_request.yaml&title=USWDS+-+Feature%3A+%5BYOUR+TITLE%5D).  A USWDS core team member may reach out to you if we need further clarification on your submission.
+
 
+### Submitting code contributions
 
-### Submitting pull requests
+#### Getting started with USWDS code
 
-If you want to propose a code fix, even better! Here’s a few steps to help you submit pull requests.
+1. First, fork this repo into your GitHub account. Read more about [forking a repo on GitHub](https://help.github.com/articles/fork-a-repo/).
+1. Open your local copy of the repository then run the following command in terminal to install project dependencies:
+    ```sh
+    npm install
+    ```
+1. Now that all of your dependencies are installed, start your local server by running the following command:
+    ```sh
+    npm start
+    ```
+1. Open `localhost:6006` in your browser to see your local build of the the USWDS component library in Storybook.
 
-#### If it’s a bug fix:
+Here are a few other utility commands you may find useful:
 
-1. Check our [open issues backlog](https://github.com/uswds/uswds/issues) for any duplicate or similar issues. 
-2. If your bug has already been submitted, feel free to comment and provide additional context (if different than the original submission).  
-3. If your proposed fix is not in the open issues backlog, create an [issue](https://github.com/uswds/uswds/issues/new/choose) for the change you’re proposing.  This helps us track our work. 
-4. Fork this repo into your GitHub account. Read more about forking a repo on [GitHub](https://help.github.com/articles/fork-a-repo/). 
+- `npm run lint`: Runs `eslint` and `sass-lint` against JavaScript and Sass files.
+- `npm run prettier`: Runs `prettier` against HTML, JavaScript, and Sass files.
+- `npm test`: Runs all tests and linters.
```

Comment:
Thanks @mahoneycm. I lean towards relying on the PR checklist to communicate this, but will leave this comment open in case anyone else wants to include it. Curious what you all think.

Reply:
Let's leave it only in the checklist for now, just to keep it simple